client/coral-framework/services/i18n.js

```diff
@@ -0,0 +1,93 @@
+const time = require('timeago.js');
```

Comment:
es6 modules are better statically analysable and enables advanced features like tree-shaking.